src/components/containers/__test__/resetRequest.test.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { mount } from "enzyme";
+import ResetRequestPage from "../resetRequest";
+
+describe("ResetRequest Component", () => {
+  it("It should open the modal", () => {
+    const resetComponent = mount(<ResetRequestPage />);
+    resetComponent.find("button.ui.button").simulate("click");
+    const formText = resetComponent.find("div.modal.active div.ui.header");
+    expect(formText.first().text()).toEqual("Send reset link");
+    resetComponent.unmount();
```

Comment:
you can use `beforeEach` and `afterEach` for this rather than repeating

Reply:
@justmesam  Thank you for the feedback. I have implemented it.